lib/chefspec/coverage.rb

```diff
@@ -28,6 +28,8 @@ def #{name}(*args, &block)
     def initialize
       @collection = {}
       @filters    = {}
+      template = ChefSpec.root.join('templates', 'coverage', 'human.erb')
```

Comment:
I think the instance variable here should really be the template itself - it doesn't make sense to construct an ERB object that might be overwritten later by the `set_template` function.